checkov/common/util/docs_generator.py

```diff
@@ -75,6 +78,9 @@ def add_from_repository(registry: Union[BaseCheckRegistry, BaseGraphRegistry], c
         graph_registry.load_checks()
         add_from_repository(graph_registry, "resource", "Terraform")
     if any(x in framework_list for x in ("all", "cloudformation")):
+        graph_registry = get_graph_checks_registry("cloudformation")
```

Comment:
Why is this part necessary only for CFN?

Reply:
I think we should add it to Kubernetes and Bicep too